src/sonic-yang-models/yang-models/sonic-acl.yang

```diff
@@ -139,7 +151,7 @@ module sonic-acl {
 
 				leaf ETHER_TYPE {
 					type string {
-						pattern "(0x88CC|0x8100|0x8915|0x0806|0x0800|0x86DD|0x8847)";
+					    pattern "0x[6-9a-fA-F][0-9a-fA-F]{2,3}|0x[1-9a-fA-F][0-9a-fA-F]{3,4}";
```

Comment:
Why the ```ETHER_TYPE``` filed is updated?

Reply:
For L2ACL user can specify any valid ethertype for the L2 traffic filtering.

Reply:
I see. Thanks